configs/bahmni/frontend/openmrs/apps/reports/reports.json

```diff
@@ -56,6 +56,45 @@
             "showTotalColumn":true
         }
     },
+    "pivotComplaint": {
+        "name": "Total Chief Complaints",
+        "type": "aggregation",
+        "config": {
+            "report":{
+                "type": "observations",
+                "config": {
+                    "conceptNamesToFilter":["Chief Complaint","Cambodia_Chief Complaint"],
+                    "conceptNameDisplayFormat": "shortNamePreferred",
+                    "ageGroupName": "Adult Child"
+                }
+            },
+            "rowGroups": [       "Concept Name", "value"        ],
+            "columnGroups": [      "Location Name" ,       "Adult Child" , "Gender"       ],
+            "distinctGroups": [    "Obs Id"    ],
```

Comment:
Format is unusual (w/ thoses spaces before and after array member).

Reply:
I just imported the report as it was, and it closely resembles the format of the other reports in this JSON file. I suggest adding the JSON formatting in a separate commit.